Doc/library/ctypes.rst

```diff
@@ -482,9 +483,11 @@ a string pointer and a char, and returns a pointer to a string::
    None
    >>>
 
-If you want to avoid the ``ord("x")`` calls above, you can set the
-:attr:`argtypes` attribute, and the second argument will be converted from a
-single character Python bytes object into a C char::
+If you want to avoid the :func:`ord("x") <ord>` calls above, you can set the
+:attr:`~_FuncPtr.argtypes` attribute, and the second argument will be converted from a
+single character Python bytes object into a C char:
+
+.. doctest::
 
    >>> strchr.restype = c_char_p
```

Comment:
It seems that the lack of `testsetup` caused the test to fail here. See **main branch:** https://github.com/python/cpython/blob/a6675b1a597c67be972598ac8562883fabe48099/Doc/library/ctypes.rst?plain=1#L465-L475

Reply:
I'm not sure it is worth the hassle to fix this backport.